client/client.go

```diff
@@ -41,7 +41,7 @@ var (
 	ErrRetriable = errors.New("retriable http status code received")
 )
 
-// APIClient manages communication with the Rosetta API v1.4.12
+// APIClient manages communication with the Rosetta API v1.4.13
```

Comment:
this should probably be `1.4.15`?

Reply:
That's generated via `codegen.sh` in the spec repo and is based off the value in `api.json`

The release PR did not include an update to that value.

Reply:
> The release PR did not include an update to that value.

Should it? or is that version a different thing

Reply:
With the exception of 1.4.14, it appears the other releases included the version bump in `api.json`